src/file/file.rpc.ts

```diff
@@ -19,52 +26,72 @@ const packageDefinition = protoLoader.loadSync(
     oneofs: true,
   });
 
+// Has the full package hierarchy
+const protoDescriptor = grpc.loadPackageDefinition(packageDefinition);
+const file_proto : any = protoDescriptor.file;
+
 export const serviceNames: string[] = ['', 'file.fileService'];
 export const healthCheckStatusMap = {
   '': HealthCheckResponse.ServingStatus.UNKNOWN,
   serviceName: HealthCheckResponse.ServingStatus.UNKNOWN
 };
 
-// Has the full package hierarchy
-const protoDescriptor = grpc.loadPackageDefinition(packageDefinition);
-const file_proto = protoDescriptor.file;
-
 /**
  * The FileServer class, containing all of the FileServer methods.
  */
 export class FileServer {
   public server: any;
   public grpcHealthCheck: GrpcHealthCheck;
+
   public constructor(port: string) {
     this.server = new grpc.Server();
+    this.addServices();
+    this.server.bind(`0.0.0.0:${port}`, grpc.ServerCredentials.createInsecure());
+  }
 
+  private addServices () {
     // Register the health service
     this.grpcHealthCheck = new GrpcHealthCheck(healthCheckStatusMap);
     this.server.addService(HealthService, this.grpcHealthCheck);
 
-    this.server.addService(file_proto.FileService.service, {
-      GenerateKey: this.generateKey,
-      CreateUpload: this.createUpload,
-      UpdateUploadID: this.updateUpload,
-      GetUploadByID: this.getUploadByID,
-      DeleteUploadByID: this.deleteUploadByID,
-      GetFileByID: this.getFileByID,
-      GetFileByKey: this.getFileByKey,
-      GetFilesByFolder: this.getFilesByFolder,
-      CreateFile: this.createFile,
-      DeleteFile: this.deleteFile,
-      IsAllowed: this.isAllowed,
-    });
+    const services = {
```

Comment:
`services` should be `service`.

Reply:
doesn't sound right. choose another name

Reply:
It is right.
This object represent the service ( multiple functions that together assembles the service)
Therefore, the variable should be named `service`

Reply:
But these are not the only functions in the service. there is also the healthcheck, and there might be more

Reply:
how about 'serviceMethods'?

Reply:
@Shahar-Y No. Just call this `service` or even `fileService`, these are the implemented functions that `FileService` defines in the protobuf.

Reply:
Done.

Reply:
@Shahar-Y No.